docker/Dockerfile.dev

```diff
@@ -83,7 +110,6 @@ RUN python -m pip install -U pip setuptools \
     && chown -R model-server /home/venv
 
 EXPOSE 8080 8081 8082 7070 7071
-USER model-server
```

Comment:
What's the reason for removing this?

Reply:
For development image it was more convenient to be able to re-install python packages. However, does not make sense for upstreaming. Thanks for pointing it out!

Reply:
Sounds good @samutamm, it's also definitely within our right to use our own rocm Dockerfile, so if there is a need to do this for some rocm specific changes we can go with that approach.

Reply:
Agree, having separate rocm Dockerfile is also an option. This change however is not rocm specific..